src/engine/effects/engineeffectsdelay.cpp

```diff
@@ -36,7 +36,9 @@ void EngineEffectsDelay::process(CSAMPLE* pInOut,
         // Read the samples using the previous group delay samples.
         m_pDelayBuffer->read(inOutSpan, m_prevDelaySamples);
         // Read the samples using the current group delay samples.
-        m_pDelayBuffer->read(m_currentDelayBuffer.subspan(0, iBufferSize), m_currentDelaySamples);
+        m_pDelayBuffer->read(
+                m_currentDelayBuffer.span().subspan(0, iBufferSize),
```

Comment:
```suggestion
                m_currentDelayBuffer.span().first(iBufferSize),
```

Reply:
Should be done with a fix for your previous proposal for the same part of the code. I mean the version using the `tmpBufferView `variable and primarily `inOutSpan.size()` for the size. IMO a more solid solution preventing errors in case of code changes.